### ISSUE TITLE
[EN] translations

### ISSUE THREAD

**Issue by vmxt** [koikiss-dev/jimov_api#110]
Looks like the API is good on my end the details were in english but it's better to provide ENG sub/translation for both providers (anime & manga)

**Comment by Mawfyy** [comment]
You're right, I'll see later when I get time, thanks for sharing that. <3
